rfcs/0097-no-read-store-dir.md

```diff
@@ -0,0 +1,89 @@
+---
+feature: no-read-store-dir
+start-date: 2021-07-04
+author: Las Safin
+co-authors:
+shepherd-team:
+shepherd-leader:
+related-issues:
+---
+
+# Summary
+[summary]: #summary
+
+Set the permissions for /nix/store to 1771 instead of 1775, disabling reading the directory for other users.
```

Comment:
Don't forget about `/nix/store/.links` (The storage location for optimized files) and `/nix/store/trash` (Used to stage directories for GC).

Reply:
Thanks.